### PR TITLE
Time PHEP

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: pheps/phep-9999.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+However, all of the core PyHC packages were created not only before the existence of PyHC.
+But also before many core libraries existed within the wider Scientiﬁc Python ecosystem (from `numpy` to `scipy` to `astropy` to `spiceypy` etc) meant that all core PyHC packages had to create their own implementations of time (and other components like coordinates) due to necessity.
+
+Now that upstream ecosystem has grown at a rapid pace, the need for PyHC core package specific implementations has reduced.
+This actually covers at least three areas (time, coordinate and units), however the focus for this PHEP is to start at time as that is considered the "simplest" case for standardization.
+
+# Rationale
+
+One of the main focuses within PyHC should be around the goal of reducing the amount of code that each package has to carry around and maintain to do exactly the same.
```
Comment (Cadair) [inline pheps/phep-9999.md:39]:
```suggestion
One of the main focuses within PyHC should be around the goal of reducing the amount of code that each package has to carry around and maintain to do implement the same functionality.
```

File: pheps/phep-9999.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+This means that the limited time that maintainers or library authors have to focus on their package should be around adding more interesting topics like new features.
```
Comment (Cadair) [inline pheps/phep-9999.md:40]:
```suggestion
This means that the limited time that maintainers or library authors have to focus on their package would be able to be used for things like adding new heliophysics specific features.
```

File: pheps/phep-9999.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+This makes the usage of `astropy` very ideal as they are backed with a larger community as well as core team that maintains the library.
```
Comment (Cadair) [inline pheps/phep-9999.md:41]:
```suggestion
This makes the usage of `astropy` ideal as they are backed by a larger community as well as core team that maintains the library, and are well funded.
```

File: pheps/phep-9999.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+It already is the foundation for many packages within astronomy and provides a strong overlap with existing objectives of many packages within PyHC.
+
+Current community objections are:
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply pheps/phep-9999.md]: See if what I added helps.

File: pheps/phep-9999.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+- This would require funding and that might be difficult based on the currently available sources.
+- Potential lack of innovation if packages use the same framework.
+- Education of a new time object for existing users of PyHC packages.
+
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply pheps/phep-9999.md:66]: Yeah, I was stumped on this.

**Pull request merged**